pulumi/aws/anthos/ingress/boa-ingress.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: bankofanthos
+spec:
+  ingressClassName: nginx # use only with k8s version >= 1.18.0
+  rules:
+  - host: demo.example.com
```

Comment:
Doing something with this line would be good. Perhaps commenting it.

Reply:
This file has been removed and the logic built into the plumi code.